frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -386,6 +391,10 @@ export const CreateVMWizardPageComponent: React.FC<CreateVMWizardPageComponentPr
     );
   }
 
+  const searchParams = new URLSearchParams(search);
+  const userMode = searchParams.get('mode');
```

Comment:
```suggestion
  const userMode = searchParams.get('mode') || VMWizardMode.VM;
```

IMO we should keep the default route as well so the user can access the page with easier url